person.rb

```diff
@@ -35,9 +35,9 @@ def can_use_services?
   private :of_age?
 end
 
-person = Person.new(22, 'maximilianus')
-puts person.correct_name
-capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalizedPerson)
-puts capitalized_trimmed_person.correct_name
+# person = Person.new(22, 'maximilianus')
+# puts person.correct_name
+# capitalized_person = CapitalizeDecorator.new(person)
+# puts capitalized_person.correct_name
+# capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
+# puts capitalized_trimmed_person.correct_name
```

Comment:
- [ ] For best practices reasons and clean code, Kindly ensure that you remove all the commented and unused code lines from your code base to make it more professional.